terraform/firewall/outputs.tf

```diff
@@ -0,0 +1,6 @@
+output "id" {
+  value = (
+    length(azurerm_firewall.adl_afw) > 0 ?
+    azurerm_firewall.adl_afw[0].id : ""
+  )
+}
```

Comment:
Consistent with other modules, let's output name and resource_group_name:

`output "name" {
  value = (
    length(azurerm_firewall.adl_afw) > 0 ?
    azurerm_firewall.adl_afw[0].name : ""
  )
}`
`output "resource_group_name" {
  value = (
    length(azurerm_firewall.adl_afw) > 0 ?
    azurerm_firewall.adl_afw[0].resource_group_name : ""
  )
}`

Reply:
Just implemented suggested changes. Resolving conversation. Re-open conversation if you have further suggestions or disagree with the changes proposed and implemented.